src/main/webapp/resources/js/pages/analyses/analysis-outputs-table.js

```diff
@@ -314,27 +317,41 @@ async function getTableData(isShared = true) {
             ? i18n("analysis.table.name.unknown")
             : wfInfo.name;
         return `${name} (${version})`;
-      }
+      },
     },
     {
       field: "analysisSubmissionName",
+      filter: "agTextColumnFilter",
       headerName: i18n("analysis-submission"),
-      cellRenderer: p =>
-        `<a href="${BASE_URL}analysis/${p.data.analysisSubmissionId}" target="_blank">${p.data.analysisSubmissionName}</a>`
+      cellRenderer: (p) =>
+        `<a href="${BASE_URL}analysis/${p.data.analysisSubmissionId}" target="_blank">${p.data.analysisSubmissionName}</a>`,
     },
     PROJECT_ID
       ? {
           field: "userId",
           headerName: i18n("project.export.submitter"),
-          cellRenderer: p => `${p.data.userFirstName} ${p.data.userLastName}`
+          cellRenderer: (p) => `${p.data.userFirstName} ${p.data.userLastName}`,
         }
       : null,
     {
       field: "createdDate",
       headerName: i18n("analysis.date-created"),
-      cellRenderer: p => formatDate({ date: p.data.createdDate })
-    }
-  ].filter(header => header !== null);
+      filter: "agDateColumnFilter",
```

Comment:
The date filter is not working correctly.

`Equals` Not working at all
`Greater Than` Works but will also work for the `than` value. Say for example Greater than February 11th...It will also return the results for February 11th
`Not Equal` works but also returns the results for the date you want to exclude